presets/wind/button/index.js

```diff
@@ -14,7 +14,7 @@ export default {
                 'px-3 py-2': props.size === 'large'
             },
             {
-                'h-8 w-8 p-0': props.label == null && props.icon !== null
+                'h-8 w-8 p-0': props.label == null && props.icon !== null && props.size !== 'small'
```

Comment:
I'm not using wind, but to match my other suggestion, we'd let size deal with the y-padding:
```suggestion
                'h-8 w-8 px-0': props.label == null && props.icon !== null
```